scripts/unity-utils.ps1

```diff
@@ -0,0 +1,149 @@
+$RunUnityLicenseRetryTimeoutSeconds = 3600
```

Comment:
This file is a dupe of scripts/unity-utils.ps1
Do we need a copy? Can we use a symlink?

Reply:
neither, one was unused, don't really know how that happened... removed